ci/Jenkinsfile.tests

```diff
@@ -5,12 +5,12 @@ library 'status-jenkins-lib@v1.9.13'
 def isPRBuild = utils.isPRBuild()
 
 pipeline {
-  agent { label 'linux && x86_64 && nix-2.19' }
+  agent { label 'linux && x86_64 && nix-2.24' }
 
   options {
     timestamps()
     /* Prevent Jenkins jobs from running forever */
-    timeout(time: 20, unit: 'MINUTES')
+    timeout(time: 60, unit: 'MINUTES')
```

Comment:
Why tho?

Reply:
some CI jobs were timing out in 1st few pipeline runs hence timeout extension was required.
This is not required anymore.